cmd/scheduler/app/options/options.go

```diff
@@ -45,6 +45,9 @@ type Options struct {
 	SchedulerEstimatorTimeout metav1.Duration
 	// SchedulerEstimatorPort is the port that the accurate scheduler estimator server serves at.
 	SchedulerEstimatorPort int
+
+	//EnablePropagateEmptyWorkLoad allow workload with replicas 0 propagated to member clusters
```

Comment:
`// EnablePropagateEmptyWorkLoad represents whether workload with 0 replicas could be propagated to member clusters.`